packages/checklist/pubspec.yaml

```diff
@@ -1,7 +1,8 @@
 name: checklist
 description: "A new Flutter package project."
 version: 0.0.1
-#homepage:
+homepage: "https://github.com/egovernments/health-campaign-field-worker-app/tree/checklist-tanishi/packages/checklist"
```

Comment:
https://github.com/egovernments/health-campaign-field-worker-app/tree/checklist-package

should be this

Reply:
> https://github.com/egovernments/health-campaign-field-worker-app/tree/checklist-package
> 
> should be this

okayy